include/krml/internal/target.h

```diff
@@ -57,6 +57,18 @@
 #  define KRML_HOST_IGNORE(x) (void)(x)
 #endif
 
+#ifndef KRML_NOINLINE
+#  if defined(_MSC_VER)
+#    define KRML_NOINLINE __declspec(noinline)
+#  elif defined (__GNUC__)
+#    define KRML_NOINLINE __attribute__((noinline))
+#  else
+#    warning "The KRML_NOINLINE macro is not defined for this toolchain!"
```

Comment:
In this third case, we could simply define `KRML_NOINLINE` to be empty with the warning?

Reply:
done